posts/react-compound-components/2021-5-25-React-compound-compnents.md

```diff
@@ -0,0 +1,250 @@
+---
+templateKey: "blog-post"
+title: "React compound components (Hooks + typescript)"
+date: 2021-05-25
+featuredpost: false
+description: >-
+  This article talks about what are compound components and how to build an accordion component using the concept of compound components.
```

Comment:
This article explains with an example what compound components in react are . How to build a compound component with a tutorial to build accordion